pkg/commands/git.go

```diff
@@ -274,40 +285,40 @@ func (c *GitCommand) AbortMerge() error {
 	return c.OSCommand.RunCommand("git merge --abort")
 }
 
-// UsingGpg tells us whether the user has gpg enabled so that we can know
+// usingGpg tells us whether the user has gpg enabled so that we can know
 // whether we need to run a subprocess to allow them to enter their password
-func (c *GitCommand) UsingGpg() bool {
-	gpgsign, _ := gitconfig.Global("commit.gpgsign")
-	if gpgsign == "" {
-		gpgsign, _ = gitconfig.Local("commit.gpgsign")
```

Comment:
Why are we removing the fallback to the local gpgsign value? (btw while reading this it appears we actually should be first trying the local one then falling back to the global one)

Reply:
Ah shit, I guess I was tired yesterday, I read twice the same function called.